### DISCUSSION TITLE
Galaxy Brain Achievement1

### DISCUSSION THREAD

**Discussion by DSSTester** [Schweinepriester/github-profile-achievements#297] [Q&A] [answered]
testing testing testing
testing testing testing
testing testing testing
testing testing testing

**Comment by PaulSpoerry** [comment, marked as answer]
Is this the answer?

**Reply by csthenry** [reply]
I think I did the same thing as you, but I still didn't get the Galaxy Brain Achievement.

**Comment by ghost** [comment]
> testing testing testing testing testing testing testing testing testing testing testing testing

Is this the answer?
